twilio/twilio-flex-shim.js

```diff
@@ -0,0 +1,91 @@
+var $callData = null;
+function isTwilioPC(pcConfig) {
+  console.warn('isTwilioPC ', pcConfig);
```

Comment:
Are we going to keep all console messages for this version ?

Reply:
Thanks, will remove it.